### ISSUE TITLE
Consider adding timestamps in ms format

### ISSUE THREAD

**Issue by kjnilsson** [rabbitmq/rabbitmq-shovel#28]
currently the timestamp added is only using [seconds resolution](https://github.com/rabbitmq/rabbitmq-shovel/blob/master/src/rabbit_shovel_util.erl#L41).

**Comment by michaelklishin** [comment]
Same idea as in https://github.com/rabbitmq/rabbitmq-event-exchange/issues/12.

**Comment by kjnilsson** [comment]
Yes although here we already use a custom header so we can either update the undocumented custom header or add another custom header. I presume someone must have asked for it at some point so adding another header may be safer.

**Comment by michaelklishin** [comment]
Agreed.
